cmd/client.go

```diff
@@ -50,42 +67,43 @@ type CloudproberClient struct {
 //	  - nil: No error occurred and the client was successfully initialised.
 //	  - gRPC Error: See https://godoc.org/google.golang.org/grpc/codes for error codes.
 //		-> See https://github.com/grpc/grpc-go/blob/d25c71b54334380ff1febd25d88064b36de44b3c/clientconn.go#L123
-func (cpc *CloudproberClient) InitClient(rpcServer string) error {
-	cpc.clientMux.Lock()
-	defer cpc.clientMux.Unlock()
+func (client *CloudproberClient) initClient(rpcServer string) error {
```

Comment:
Could we just integrate this into `NewClient`, since I don't think it should ever need to be called elsewhere?

Reply:
Ack.